__tests__/components/editor/ImportResourceTemplate.test.js

```diff
@@ -124,44 +124,125 @@ describe('<ImportResourceTemplate />', () => {
 
       expect(wrapper.state().message).toEqual([])
 
-      wrapper.instance().updateStateFromServerResponse({ status: 200, headers: {} })
+      wrapper.instance().updateStateFromServerResponse({ status: 200, headers: {} }, 0)
 
       expect(wrapper.state().message).toEqual(['Unexpected response (200)! '])
     })
+
     it('sets modalShow to true when receiving HTTP 409 and errors >= profileCount', () => {
       // Set new wrapper in each of these tests because we are changing state
       wrapper = shallow(<ImportResourceTemplate.WrappedComponent authenticationState={authenticationState}/>)
 
       expect(wrapper.state().modalShow).toBe(false)
 
       wrapper.instance().updateStateFromServerResponse({ status: 409 , headers: {} }, 0)
+
       expect(wrapper.state().modalShow).toBe(true)
     })
+
     it('sets message in state with any create operation not resulting in HTTP 409', () => {
       // Set new wrapper in each of these tests because we are changing state
       wrapper = shallow(<ImportResourceTemplate.WrappedComponent authenticationState={authenticationState}/>)
 
       expect(wrapper.state().message).toEqual([])
 
-      wrapper.instance().updateStateFromServerResponse({ status: 201 , headers: { location: 'http://foo.bar' } })
+      wrapper.instance().updateStateFromServerResponse({ status: 201 , headers: { location: 'http://foo.bar' } }, 0)
 
       expect(wrapper.state().message).toEqual(['Created http://foo.bar'])
     })
+
+    it('resets createResourceError and message in component state when response is OK', () => {
+      // Set new wrapper in each of these tests because we are changing state
+      wrapper = shallow(<ImportResourceTemplate.WrappedComponent authenticationState={authenticationState}/>)
+
+      wrapper.setState({ createResourceError: ['foo'], message: ['bar'] })
+      expect(wrapper.state().createResourceError).toEqual(['foo'])
+      expect(wrapper.state().message).toEqual(['bar'])
+
+      wrapper.instance().updateStateFromServerResponse({ ok: true, status: 204 }, 0)
+
+      expect(wrapper.state().createResourceError).toEqual([])
+      expect(wrapper.state().message).toEqual(['Updated '])
+    })
+
+    it('leaves createResourceError alone and appends message in component state when response is not OK', () => {
+      // Set new wrapper in each of these tests because we are changing state
+      wrapper = shallow(<ImportResourceTemplate.WrappedComponent authenticationState={authenticationState}/>)
+
+      wrapper.setState({ createResourceError: ['foo'], message: ['bar'] })
```

Comment:
Non-blocking comment, but I find using placeholder variable values like `foo` and `bar` to detract from what you're trying to do here. To me, example values you would expect makes it easier to tie back to what the code is actually trying to do.

Reply:
you mean 'bar' could be 'I am an error message' or some such?  I'd be in favor of that.

Reply:
Changed and pushed! Resolving.